x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.test.tsx

```diff
@@ -635,9 +642,9 @@ describe('when on the endpoint list page', () => {
 
     it('should show the flyout', async () => {
```

Comment:
```suggestion
    it('should show the flyout and footer', async () => {
```

:stuck_out_tongue_closed_eyes:

Reply:
👍